services/src/main/java/org/fao/geonet/api/OpenApiController.java

```diff
@@ -76,22 +84,21 @@ public class OpenApiController extends AbstractOpenApiResource {
 
     private final Optional<SecurityOAuth2Provider> springSecurityOAuth2Provider;
 
-    private final Optional<RouterFunctionProvider> routerFunctionProvider;
+    private final List<RouterOperation> routerOperations;
 
     @Autowired
     public OpenApiController(ObjectFactory<OpenAPIService> openAPIBuilderObjectFactory,
                              AbstractRequestService requestBuilder,
                              GenericResponseService responseBuilder,
                              OperationService operationParser,
                              RequestMappingInfoHandlerMapping requestMappingHandlerMapping,
-                             Optional<ActuatorProvider> servletContextProvider,
-                             Optional<List<OperationCustomizer>> operationCustomizers,
-                             Optional<List<OpenApiCustomiser>> openApiCustomisers,
                              SpringDocConfigProperties springDocConfigProperties,
-                             Optional<ActuatorProvider> actuatorProvider,
-                             Optional<SecurityOAuth2Provider> springSecurityOAuth2Provider,
-                             Optional<RouterFunctionProvider> routerFunctionProvider) {
-        super(DEFAULT_GROUP_NAME, openAPIBuilderObjectFactory, requestBuilder, responseBuilder, operationParser, operationCustomizers, openApiCustomisers, springDocConfigProperties, actuatorProvider);
+                             SpringDocProviders springDocProviders,
+                             SpringDocCustomizers springDocCustomizers,
+                             List<RouterOperation> routerOperations,
+                             Optional<ActuatorProvider> servletContextProvider,
+                             Optional<SecurityOAuth2Provider> springSecurityOAuth2Provider) {
+        super(DEFAULT_GROUP_NAME, openAPIBuilderObjectFactory, requestBuilder, responseBuilder, operationParser, springDocConfigProperties, springDocProviders, springDocCustomizers);
         springDocConfigProperties.setPathsToExclude(Arrays.asList(new String[]{"/0.1/**"}));
```

Comment:
```suggestion
        springDocConfigProperties.setPathsToExclude(Arrays.asList("/0.1/**"));
```

Reply:
@ianwallen the next line GitHub doesn't allow to make a suggestion, change, but a similar change can be applied:

```java
springDocConfigProperties.setPackagesToScan(Arrays.asList("org.fao.geonet.api",
                "org.fao.geonet.services.inspireatom",
                "org.fao.geonet.monitor.service"));
```

Reply:
I will use `List.of("/0.1/**")` instead of your suggestion since it is only a single item.